yoti_python_sdk/http.py

```diff
@@ -24,12 +24,38 @@
 
 
 class YotiResponse(object):
-    def __init__(self, status_code, text):
+    def __init__(self, status_code, text, headers=None, content=None):
+        if headers is None:
+            headers = {}
+
         self.status_code = status_code
         self.text = text
+        self.content = content
+        self.headers = headers
+
 
+class MediaResponse(object):
+    def __init__(self, response):
+        self.response = response
+
+    @property
+    def status_code(self):
+        return self.response.status_code
 
-class RequestHandler:
+    @property
+    def content(self):
+        return self.response.content
+
+    @property
+    def text(self):
+        return self.response.text
+
+    @property
+    def headers(self):
+        return self.response.headers
```

Comment:
Can we now remove `MediaResponse` as `YotiResponse` supports binary content?

Reply:
Fixed in a7259b8